README.md

```diff
@@ -58,6 +39,13 @@ Please check out our tutorials:
 Please take a look at [these slides](https://www.slideshare.net/BogdanRoman1/bluebrain-nexus-technical-introduction-91266871)
 for a short technical introduction to Nexus.
 
+# Participation
+
+There are several channels provided to address different issues:
+- **Feature request**: If there is a feature you would like us to implement and it is not listed on the [feature issues](https://github.com/BlueBrain/nexus/issues?q=is%3Aopen+is%3Aissue+label%3Afeature), create a [github issue](https://github.com/BlueBrain/nexus/issues/new?title=[Feature]%20My%20feature%20request) describing your feature with as much detail as possible and tag it with the `feature` label.
```

Comment:
I would word it a bit differently:
"If there is a feature you would like to see in Blue Brain Nexus, please first consult the list of [open feature requests](https://github.com/BlueBrain/nexus/issues?q=is%3Aopen+is%3Aissue+label%3Afeature). In case there isn't already one, please open a [feature request](https://github.com/BlueBrain/nexus/issues/new?title=[Feature]%20My%20feature%20request&labels=feature) describing your feature with as much detail as possible."

Reply:
Please note the feature request links auto-add the "feature" tag to the issue